charts/zeebe-benchmark/values.yaml

```diff
@@ -525,7 +525,7 @@ camunda-platform:
         # Persistent Volume Storage Class
         storageClass: "ssd"
         # Persistent Volume Size
-        size: 16Gi
+        size: 32Gi
```

Comment:
I've seen it require even more...let's give it 64GB for now, just to be safe. We can scale down later based on experience.